thumbprint/proguard-rules.pro

```diff
@@ -1,11 +1,13 @@
 # For more details, see
 #   http://developer.android.com/guide/developing/tools/proguard.html
 
-# Add any project specific keep options here:
+# leep all public classes, otherwise they will be removed by tree-shaking since they may be unused within this library
```

Comment:
Nit: Spelling "keep" as "leep"

Reply:
This is actually a highly technical proguard term, but I suppose I can switch it to the common spelling to avoid confusion

Reply:
Oh if it's a proguard term feel free to leave it